modules/custom-css/custom-css-4.7.php

```diff
@@ -0,0 +1,498 @@
+<?php
+
+/**
+ * Class Jetpack_Custom_CSS_Enhancements
+ */
+class Jetpack_Custom_CSS_Enhancements {
+	public static function add_hooks() {
+		add_action( 'init', array( __CLASS__, 'init' ) );
+		add_action( 'admin_menu', array( __CLASS__, 'admin_menu' ) );
+		add_action( 'customize_controls_enqueue_scripts', array( __CLASS__, 'customize_controls_enqueue_scripts' ) );
+		add_action( 'customize_register', array( __CLASS__, 'customize_register' ) );
+		add_filter( 'map_meta_cap', array( __CLASS__, 'map_meta_cap' ), 20, 2 );
+
+		// Stuff for stripping out the theme's default stylesheet...
+		add_filter( 'stylesheet_uri', array( __CLASS__, 'style_filter' ) );
+		add_filter( 'safecss_skip_stylesheet', array( __CLASS__, 'preview_skip_stylesheet' ) );
+
+		// Stuff for overriding content width...
+		add_action( 'init', array( __CLASS__, 'preview_content_width' ) );
```

Comment:
This action, and any others that are supposed to only run in the customizer preview, should only be added in the `customize_preview_init` hook.